docs/api/shallow.md

```diff
@@ -91,7 +91,7 @@ Returns a string representation of the text nodes in the current render tree.
 #### [`.html() => String`](ShallowWrapper/html.md)
 Returns a static HTML rendering of the current node.
 
-#### [`.get(index) => ShallowWrapper`](ShallowWrapper/get.md)
+#### [`.get(index) => ReactElement`](ShallowWrapper/get.md)
```

Comment:
I don't think so. `get()` is still a method in the `ShallowWrapper` API, after all.